kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -332,6 +335,12 @@
       "Whether to enable REST Proxy V3 API. Default is true.";
   private static final boolean API_V3_ENABLE_DEFAULT = true;
 
+  public static final String SCHEMA_CACHE_SPEC_CONFIG = "schema.cache.spec";
+  private static final String SCHEMA_CACHE_SPEC_DOC =
+      "Guava CacheBuilderSpec string for the schema cache. Default is"
+          + "'maximumSize=10000,expireAfterAccess=10m'";
+  private static final String SCHEMA_CACHE_SPEC_DEFAULT = "maximumSize=10000,expireAfterAccess=10m";
```

Comment:
this is kind of leaking internal details (though not in a very serious way). i'd probably have two separate properties `SCHEMA_CACHE_MAX_SIZE_BYTES` and `SCHEMA_CACHE_EXPIRE_AFTER_MINUTES`.

Reply:
i realize that is less flexible, but i don't that matters.

Reply:
Obsolete.

Reply:
great. if it's obsolete though, i guess the question becomes why is it still here?

Reply:
Forgot to remove this one. :)